mappings/net/minecraft/client/toast/Toast.mapping

```diff
@@ -1,9 +1,10 @@
 CLASS net/minecraft/unmapped/C_zqhkdxaf net/minecraft/client/toast/Toast
-	FIELD f_msurgqxf TYPE Ljava/lang/Object;
+	FIELD f_msurgqxf ID Ljava/lang/Object;
```

Comment:
```suggestion
	FIELD f_msurgqxf DEFAULT_ID Ljava/lang/Object;
```

Reply:
Mojmap actually suggests that this weird Id is actually the id that is used on toasts that don't have the same system that `SystemToast` has and only really has one id that is used! Such as `RecipeToast`; It calls it `NO_TOKEN`, but i don't think it's a good name?

Reply:
oh well, placeholder name for now..? there should be a better name though

Reply:
Do non-`SystemToast` toasts not override each other (afaict they all use this default id)?

I thought they did based on how `ToastManager#getToast` works.

Reply:
So, apparently they avoid this by using different classes alongside the ids? I still have to look at this more closely (something that is now impossible thanks to 24w14a branch being broken :p)

Reply:
But then they still override toasts of the same class (because they all have the same id).

Maybe this should be `NO_ID` with javadoc
> The default identifier used by toast classes for which only one instance is displayed at a time.

And a javadoc at `getId` could be something like
> Toasts with the same class with different identifiers can co-exist, while toasts sharing the same class and identifier one will overwrite each other.

(mostly copied from your javadoc)

Reply:
Ah fair; this matches with Mojang's intention of `NO_TOKEN` perfectly while not being confusing at the same time; I'll investigate this tomorrow, since a broken Enigma complicates things, and yeah, if I have to workaround, I'd rather work around it tomorrow morning :p

(btw, `getId` is once again a silly target, I'd find an another more appropriate method; clarifying Javadocs, however, are the correct path for this)

Reply:
I applied the `NO_ID` suggestion; however, the Javadoc should be done on another tackle on unmapped Toast stuff; it needs to be thought about carefully, but it should be worth it